mysql-test/suite/rocksdb/r/rocksdb_set_var_extension.result

```diff
@@ -177,9 +177,9 @@ SET_VAR(rocksdb_master_skip_tx_api=0)
 @before_val	@hint_val	@after_val
 OFF	OFF	OFF
 hint_str
-SET_VAR(rocksdb_perf_context_level=5)
+SET_VAR(rocksdb_perf_context_level=4)
 @before_val	@hint_val	@after_val
-0	5	0
+0	4	0
```

Comment:
please raise doc issue to correct this:
https://www.percona.com/doc/percona-server/8.0/myrocks/variables.html#rocksdb_perf_context_level

Reply:
https://jira.percona.com/browse/PS-8093